content/threads/weekly-thread-2022-05.md

```diff
@@ -0,0 +1,11 @@
+---
+slug: weekly-thread-2022-05
+title: 'Weekly thread #05 of 2022'
+tweet_id: '1489620159585153026'
+author_username: archtechx
+created_at: 2022-02-17T16:16:06+00:00
```

Comment:
The threads are both on 17/2, but the date should be the same as on Twitter

Reply:
Also, all the tweets seem to be either on 4/2 or 16/2? I think I asked about this via chat

Reply:
> The threads are both on 17/2, but the date should be the same as on Twitter

I see, I'll fix it.

> Also, all the tweets seem to be either on 4/2 or 16/2? I think I asked about this via chat

Yeah, the #5 thread was posted on 4/2 and #6 was posted on 16/2. I think the tweet `created_at` timestamps are correct (but the thread's `created_at` is not).